webpack/build.babel.js

```diff
@@ -17,7 +17,19 @@ export default {
         test: /\.scss$/,
         use: ExtractTextPlugin.extract({
           fallback: 'style-loader',
-          use: ['css-loader', 'sass-loader', 'autoprefixer-loader'],
+          use: [
+            'css-loader',
+            {
+              loader: 'postcss-loader',
+              options: {
+                ident: 'postcss',
+                plugins: (loader) => [
+                  require('autoprefixer')()
```

Comment:
should probably need to add it as a devDep too

Reply:
I added it in `webpack/dev.js`. Is that what you meant?